cmd/apprepository-controller-new/go.mod

```diff
@@ -0,0 +1,74 @@
+module github.com/vmware-tanzu/kubeapps
+
+go 1.17
```

Comment:
I guess this file should be at the root level instead (for the sake of consistency with the rest of the go code we have)